python/hsfs/feature_store.py

```diff
@@ -604,13 +605,15 @@ def create_training_dataset(
                 the training dataset. When replaying a `Query` during model inference,
                 the label features can be omitted from the feature vector retrieval.
                 Defaults to `[]`, no label.
-            transformation_functions: A dictionary mapping tansformation functions to
+            transformation_functions: A dictionary mapping transformation functions to
                 to the features they should be applied to before writing out the
                 training data and at inference time. Defaults to `{}`, no
                 transformations.
             train_split: If `splits` is set, provide the name of the split that is going
                 to be used for training. The statistics of this split will be used for
                 transformation functions if necessary. Defaults to `None`.
+            split_type: If `splits` is set, provide the type of the method to split datasets. Currently `random` and
+            `time_series` types are supported. Defaults to `random`.
```

Comment:
maybe include a description about "time_series" split, it is not so intuitive until I read the code.